src/heretic/main.py

```diff
@@ -359,17 +366,25 @@ def objective_wrapper(trial: Trial) -> tuple[float, float]:
         key=lambda trial: trial.user_attrs["refusals"],
     )
 
-    choices = [
-        Choice(
-            title=(
-                f"[Trial {trial.user_attrs['index']:>3}] "
-                f"Refusals: {trial.user_attrs['refusals']:>2}/{len(evaluator.bad_prompts)}, "
-                f"KL divergence: {trial.user_attrs['kl_divergence']:.2f}"
-            ),
-            value=trial,
+    choices = []
+    for trial in best_trials:
+        if settings.direction_scope != "all":
+            direction_scope = ""
+        elif trial.user_attrs["direction_index"] is None:
+            direction_scope = "(per layer)"
+        else:
+            direction_scope = "(global)"
+        choices.append(
+            Choice(
+                title=(
+                    f"[Trial {trial.user_attrs['index']:>3}] "
+                    f"Refusals: {trial.user_attrs['refusals']:>2}/{len(evaluator.bad_prompts)}, "
+                    f"KL divergence: {trial.user_attrs['kl_divergence']:.2f} "
+                    f"{direction_scope}"
```

Comment:
What is the point of providing this information here? What can the user do with it? We don't show the other parameters.

Reply:
> though I think it would be good to display the parameters when you load a particular trial result.

I actually agree with that.